aisdc/attacks/worst_case_attack.py

```diff
@@ -791,47 +796,51 @@ def main():
         "--attack-metric-success-name",
         action="store",
         type=str,
-        default="AUC",
+        default="P_HIGHER_AUC",
         required=False,
         dest="attack_metric_success_name",
         help=(
-            "for computing attack success/failure based on metric_fail_thresh option. Default = %(default)s"
+            """for computing attack success/failure based on
+            --attack-metric-success-thresh option. Default = %(default)s"""
         ),
     )
 
     attack_parser.add_argument(
         "--attack-metric-success-thresh",
         action="store",
         type=float,
-        default=0.8,
+        default=0.05,
         required=False,
         dest="attack_metric_success_thresh",
         help=(
-            "for defining threshold value to measure failing for the metric defined by parameter fail-metric-name option. Default = %(default)f"
+            """for defining threshold value to measure attack success
+            for the metric defined by argument --fail-metric-name option. Default = %(default)f"""
         ),
     )
 
     attack_parser.add_argument(
         "--attack-metric-success-comp-type",
         action="store",
         type=str,
-        default="gte",
+        default="lte",
         required=False,
         dest="attack_metric_success_comp_type",
         help=(
-            "for computing attack success/failure based on metric_fail_thresh option. Default = %(default)s"
+            """for computing attack success/failure based on
+            --attack-metric-success-thresh option. Default = %(default)s"""
         ),
     )
 
     attack_parser.add_argument(
         "--attack-metric-success-count-thresh",
```

Comment:
in the class it's counting failures, so should this param be fail_count or similat?